crates/metaslang/bindings/src/lib.rs

```diff
@@ -17,9 +17,12 @@ use stack_graphs::graph::StackGraph;
 
 type Builder<'a, KT> = builder::Builder<'a, KT>;
 type GraphHandle = stack_graphs::arena::Handle<stack_graphs::graph::Node>;
+type FileHandle = stack_graphs::arena::Handle<stack_graphs::graph::File>;
 type CursorID = usize;
 pub struct DefinitionHandle(GraphHandle);
 
+pub const BUILT_INS_FILE_PATH: &str = "@@built-ins@@";
```

Comment:
I suggest we use "Source ID" instead of "file path", to match what we are going to end up with in the Public API .. While most of our testing/usage so far has been file paths, there is nothing that prevents the users from using remote IPFS/Github/NPM URIs, or any other raw strings instead of file paths ..

Also, I suggest adding `.sol` for syntax highlighting, in case this is used in an editor:

And last, although not blocking to this PR, I wonder if it is possible to supply this value to `Bindings` ctor, so that the `Compilation` API can make sure it is a name that doesn't conflict with anything the user might use?

```suggestion
pub const BUILT_INS_SOURCE_ID: &str = "@built-ins.sol";
```